src/webview_provider/newDocsGenPanel.ts

```diff
@@ -29,6 +29,7 @@ import { UsersService } from "../services/usersService";
 import { ConversationProvider } from "../comment_provider/conversationProvider";
 import { DbtDocsView } from "./DbtDocsView";
 import { ConversationService } from "../services/conversationService";
+import { QueryResultTab } from "./QueryResultTab";
```

Comment:
this can be part of queryResultPanel.ts. I have added DbtDocsView, because it was related to doc editor